src/pages/login/login.page.jsx

```diff
@@ -27,13 +28,18 @@ class Login extends Component {
     componentDidUpdate(prevProps, prevState) {
         if (prevProps.errorMessage !== this.props.errorMessage && this.props.errorMessage !== '') {
             toast.error(`Error 😓: ${this.props.errorMessage}`, {
-                position: toast.POSITION.TOP_CENTER,
-                onClose: () => this.props.ClearLoginError()
+                position: toast.POSITION.TOP_CENTER
             })
         }
 
         if (prevProps.isLoggedIn !== this.props.isLoggedIn && this.props.isLoggedIn) {
-            this.getUser();
+            this.props.FetchUserMe();
+        }
+
+        if (prevProps.user !== this.props.user && this.props.user) {
```

Comment:
Just curious, shouldn't you first check `this.props.user` then check for inequality? Or any specific reason for this

Reply:
no specific reason as such

Reply:
well, the first part is to check if the property has actually changed since the last time component was updated (basically so that we don't trigger this function twice for same user when some other property has changed) and the second part is to check that the current value of the property is not set to `null`, since that is the default value when no current user details are fetched.

Reply:
doing it in this order made sense since the second part doesn't need to be checked if the property hasn't changed at all

Reply:
Ah, I was thinking from the point of null check first

Reply:
I was considering from the point where both the conditions were a deciding factor.. in that case order shouldnt matter as much so my reply

Reply:
Yeah, in the end the order doesn't matter since both are equally important, just made sense to me sequence wise that way.